components/navbar.vue

```diff
@@ -32,30 +36,59 @@
               hover:bg-gray-400
             "
             href="#"
-            @click="logout"
           >
-            Logout
-          </a>
+            <div  class="flex w-25 h-10 font-semibold items-center text-15px" :class="class1">
+              <img class="w-7 rounded-full mr-1" :src='url'/>
+              {{username}}
+            </div>
+            <button class="font-semibold background-white w-25 h-10" :class="class2">
+              logout
+            </button>
+          </button>
         </div>
       </div>
     </ul>
   </nav>
 </template>
 
+
 <script>
 import { defineComponent } from '@nuxtjs/composition-api'
 
 export default defineComponent({
+  data(){
+    return{
+      name: '',
+      class1: '',
+      class2: 'hideme',
+    }
+  },
   computed: {
     auth() {
       return this.$store.getters.auth
     },
+    username: function(){
+      return this.$store.getters.name
+    },
+    url: function() {
+      return 'https://ui-avatars.com/api/?name='+this.$store.getters.name+'&background=0D8ABC&color=fff'
+    }
   },
   methods: {
     logout() {
       this.$store.commit('setToken', null)
-      this.$router.replace('/login')
+      this.$router.push('/login')
+      this.$cookies.remove('token')
+      this.$toast.info("SEE YOU SOON!!..")
+    },
+    swap1(){
+      this.class1='hideme'
+      this.class2=''
     },
+    swap2(){
+      this.class1=''
+      this.class2='hideme'
+    }
```

Comment:
better and cleaner methods could have been implemented make sure to check out [this](https://blog.manlakshya.tech/dynamic-classes-and-styles) :P